tests/.eslintrc.js

```diff
@@ -1,5 +1,12 @@
 module.exports = {
   env: {
     embertest: true
+  },
+  globals: {
```

Comment:
Make it so that eslint doesn't complain when we use ember test helpers without declaring them in our tests